src/common/environment.ts

```diff
@@ -2,4 +2,8 @@ export class Env {
     public static get() {
         return process.env.NODE_ENV;
     }
+
+    public static isDevelopment(): boolean {
```

Comment:
Can we add a test for this?  Also - did you confirm that will work as expected in web target?